docs/azure/sdk/snippets/unit-testing/NonLibrary/MockSecretClient.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Azure.Security.KeyVault.Secrets;
+using Azure;
+
+namespace UnitTestingSampleApp.NonLibrary;
+
+public sealed class MockSecretClient : SecretClient
+{
+    AsyncPageable<SecretProperties> _pageable;
+
+    // Allow a pageable to be passed in for mocking different responses
+    public MockSecretClient(AsyncPageable<SecretProperties> pageable)
+    {
+        _pageable = pageable;
+    }
+
+    public override Response<KeyVaultSecret> GetSecret(
+        string name,
+        string version = null,
+        CancellationToken cancellationToken = default)
+        => throw new NotImplementedException();
+
+    public override Task<Response<KeyVaultSecret>> GetSecretAsync(
+        string name,
+        string version = null,
+        CancellationToken cancellationToken = default)
+        => throw new NotImplementedException();
+
+    public override AsyncPageable<SecretProperties> GetPropertiesOfSecretsAsync
```

Comment:
Any reason why this wouldn't be an expression-bodied member like the other 2 methods?